### PR TITLE
docs: add basic readme structure

### PR DESCRIPTION
**Pull request by michaltomczyk28**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# IJoke (PAC1)
+
+## Why?
+
+We want to provide a convenient, modern-looking web application that allows users to read, add, and like jokes.
+
+## What?
+
+## How?
+
```
Comment (oskar6711) [inline README.md:8]: Add information about what technologies we are going to use

File: README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# IJoke (PAC1)
+
+## Why?
+
+We want to provide a convenient, modern-looking web application that allows users to read, add, and like jokes.
+
+## What?
+
```
Comment (oskar6711) [inline README.md]: Move "what" to the paragraph above since it answers both questions "Why" and "What"

**Review by oskar6711 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (michaltomczyk28) [reply README.md:8]: Good input! I forgot

**Review by oskar6711 — APPROVED** [review]
Nicely done!

**Pull request merged**